core/test/scripts/Voting.js

```diff
@@ -195,6 +195,33 @@ contract("scripts/Voting.js", function(accounts) {
     assert.equal((await voting.getPrice(identifier, time)).toString(), web3.utils.toWei("9155.05"));
   });
 
+  it("Notification on crash", async function() {
+    // Force a Solidity revert by returning an invalid hash in the soliditySha3 function.
+    const temp = web3.utils.soliditySha3;
+    web3.utils.soliditySha3 = () => {
+      return "0x0";
+    };
```

Comment:
Somewhat hacky, do you know of a better way?

Reply:
This seems reasonable to me - it's a test, so IMO hackiness is totally acceptable if it keeps things simple.